benchmarks/graphics_pipeline/main.cpp

```diff
@@ -120,15 +114,16 @@ class ProjApp
     std::array<grfx::ShaderModulePtr, kAvailablePsShaders.size()> mPsShaders;
     PerspCamera                                                   mCamera;
     float3                                                        mLightPosition = float3(10, 100, 10);
-    BenchmarkSettings                                             mBenchmarkSettings;
 
     std::vector<Material>  mMaterials;
     std::vector<Primitive> mPrimitives;
     std::vector<Object>    mObjects;
     TextureCache           mTextureCache;
 
 private:
-    std::shared_ptr<KnobCheckbox> pKnobAlphaBlend; // alpha blend functionality unimplemented
+    std::shared_ptr<KnobDropdown<std::string>> pKnobVs;
+    std::shared_ptr<KnobDropdown<std::string>> pKnobPs;
+    std::shared_ptr<KnobCheckbox>              pKnobAlphaBlend; // alpha blend functionality unimplemented
```

Comment:
Could this be removed since it's not implemented (and since we have real knobs now?)